driver/lib/driver.ts

```diff
@@ -141,6 +141,11 @@ class FlutterDriver extends BaseDriver {
     return true;
   }
 
+  public async proxyCommand (url, method, body = null) {
+    const result = await this.proxydriver.proxyCommand(url, method, body);
+    return result
+  }
```

Comment:
```suggestion
  public async proxyCommand (url, method, body = null) {
    this.clearNewCommandTimeout();
    const result = await this.proxydriver.proxyCommand(url, method, body);
    this.startNewCommandTimeout(cmd);
    return result
  }
```

to make newCommandTimeout in the proxydriver

Reply:
`startNewCommandTimeout` does not require any args. Another implementation in this repo is wrong. I'll modify it as another pr